.github/workflows/test-command.yml

```diff
@@ -149,6 +152,34 @@ jobs:
         repository: ${{ github.event.client_payload.pull_request.head.repo.full_name }}
         ref: ${{ github.event.client_payload.pull_request.head.ref }} 
 
+    # Determine requied version of terraform based on the target branch of the pull request. Then update the PATH to use it.
+    - name: "Determine required terraform version"
+      run: |
+        VERSION=$(cut -d/ -f1 <<<${BASE_REF})
+        if [[ ${VERSION} == 'master' ]]; then
+          VERSION=0.12
+        fi
+ 
+        OVERRIDE_VERSION=$(grep -Eo 'terraform/\d+\.\d+' <<<${LABELS} | cut -d/ -f2)
+
+        if [ -n "${OVERRIDE_VERSION}" ]; then
+          VERSION=${OVERRIDE_VERSION}
+          echo "Terraform ${VERSION} is required based on labels"
+        else
+          echo "Terraform ${VERSION} is required for ${BASE_REF}..."
```

Comment:
```suggestion
          echo "Terraform ${VERSION} is selected by default"
```

Reply:
It was selected based on the `BASE_REF`.  By default implies to me in a message like this:

> Terraform 0.12 is selected by default

But that's not really accurate. It was selected by "default" (perhaps), but that default was determined based on the `BASE_REF`

Reply:
OK, but it is not "required" either. If it were "required" it could not be successfully overridden.